src/api/axiosReq.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import conf from "./configapi";
+axios.defaults.baseURL = `${conf.mainUrl}${conf.apiKey}`;
+
+const getAxiosReq = async (url, parameters) => {
```

Comment:
you can use params name at this place and call on the line 12 will be like { params }